libero-config/config.js

```diff
@@ -0,0 +1,14 @@
+const coreConfig = require('./config--core');
+
+let customConfig;
+try {
+  customConfig = require('./config--custom');
```

Comment:
I think the custom config here is mutating the `config--core`. So if you were to say have a config--custom that looked liked
```js
const config = require('./config--core');

config.customThing = 'custom thing';

throw new Error();
```

When you call `generateConfig()` below and it returns the core config, the property `coreConfig.customThing` will be set on the core object. Obviously a synthetic example, since the `try{}catch` is obviously checking if the file exists, but just thought I'd point out

Reply:
Good point @stephenwf. I've refactored this in https://github.com/libero/patterns-core/pull/17/commits/829d17fbcb70ac7c045a1fe9823c5fad9d057ed8 to try to make it clearer what it's doing. Does that look better to you, or are you not comfortable with the fact of the mutation itself?